src/main/java/com/wiley/autotest/spring/SeleniumTestExecutionListener.java

```diff
@@ -433,16 +436,24 @@ private WebDriver initWebDriver(Settings settings, Configuration configuration)
                 } else if (StringUtils.equalsIgnoreCase(browserName, SAFARI_TECHNOLOGY_PREVIEW)) {
                     SeleniumHolder.setDriverName(SAFARI_TECHNOLOGY_PREVIEW);
                     SeleniumHolder.setPlatform(MAC);
-                    return safariTechnologyPreview(settings);
+                    return safariTechnologyPreview(settings, customDesiredCapabilities);
+                } else if (!customDesiredCapabilities.asMap().isEmpty()) {
+                    desiredCapabilities = customDesiredCapabilities;
```

Comment:
how is it planned to use desiredCapabilities set in this line? (can't quickly find it in code))

Reply:
got it now while refactoring this class